src/Symfony/Component/DomCrawler/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
   `Image` classes is now optional.
 * The `Crawler::children()` method will have a new `$selector` argument in version 5.0, 
   not defining it is deprecated.
+* Added return of element name (`_name`) in `extract()` method.
```

Comment:
most likely 4.3.0 :)

Reply:
Latest release is 4.1.17?
I think 4.2 is next release? Or no?

Reply:
4.2 is in feature freeze and already has its BETA1 release. As such i dont think it will be merged for 4.2 anymore, thus 4.3 if so.

Reply:
Changed